File: 2largearray.java
import java.util.*;
class Rohith{
    public static void main(String args[]){
   Scanner s=new Scanner(System.in);
   int max=Integer.MIN_VALUE;
//    int second=0;
   int second = Integer.MIN_VALUE;
   int arr[]=new int[5];
   for(int i=0;i<5;i++){
    arr[i]=s.nextInt();
   }
   s.close();
    //  Arrays.sort(arr);
   for(int i=0;i<5;i++){
    if(arr[i] >max){
        second=max;
        max=arr[i];    
    }
    else if(arr[i] >second && arr[i] != max){
        second=arr[i];
    }
   }
   System.out.println("Second largest number : "+second);
    }
}

File: 2largearray.py
#second largest number
def secondLargest(arr):
    arr.sort(reverse=True)
    return arr[1]

n=int(input("enter the size of the array"))
arr=[]
for i in range(n):
    element=int(input(f"Enter the element of {i+1} "))
    arr.append(element)
print(secondLargest(arr))